swagger-alpine/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM amd64/alpine:3.5
+
+RUN apk add --no-cache curl
+RUN curl -L https://github.com/go-swagger/go-swagger/releases/download/0.13.0/swagger_linux_amd64 -o swagger
+RUN chmod +x swagger
+ENTRYPOINT ["./swagger", "validate", "/go/src/github.com/fnproject/fn/docs/swagger.yml"]
```

Comment:
entrypoint should just be the first two args here, and then the command can take a file path:

```
docker run fnproject/swagger my/path/to/swagger.yml
```

Reply:
fixed